src/allotropy/parsers/luminex_xponent/luminex_xponent_structure.py

```diff
@@ -0,0 +1,336 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from io import StringIO
+import re
+from typing import Any, Optional
+
+from dateutil import parser
+import pandas as pd
+
+from allotropy.exceptions import AllotropeConversionError
+from allotropy.parsers.lines_reader import CsvReader
+from allotropy.parsers.utils.values import (
+    assert_not_none,
+    try_float,
+    try_float_from_series,
+    try_str_from_series,
+    try_str_from_series_or_none,
+)
+
+EMPTY_CSV_LINE = r"^,*$"
+CALIBRATION_BLOCK_HEADER = "Most Recent Calibration and Verification Results"
+TABLE_HEADER_PATTERN = "DataType:,{}"
+MINIMUM_CALIBRATION_LINE_COLS = 2
+EXPECTED_CALIBRATION_RESULT_LEN = 2
+LOCATION_REGEX = r"\d+\((?P<well_location>\d+,(?P<location_id>\w+))\)"
+
+
+@dataclass(frozen=True)
+class Header:
+    model_number: str
+    software_version: str
+    equipment_serial_number: str
+    analytical_method_identifier: str
+    method_version: str
+    experimental_data_identifier: str
+    sample_volume_setting: float
+    plate_well_count: float
+    measurement_time: str
+    detector_gain_setting: str
+    data_system_instance_identifier: str
+    analyst: Optional[str] = None
+
+    @staticmethod
+    def create(header_data: pd.DataFrame) -> Header:
+        info_row = header_data.iloc[0]
+        raw_datetime = try_str_from_series(info_row, "BatchStartTime")
+
+        return Header(
+            model_number=Header._get_model_number(header_data),
```

Comment:
We had the strucure/builder pattern in the past but changed to the `create` methods in the structure classes because it was easier to test. Having `classmethods` I think is a great point for maintainability.